specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/examples/virtualMachineExamples/VirtualMachine_Get_VM_WithInstanceView.json

```diff
@@ -0,0 +1,218 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2022-08-01",
+    "vmName": "myVM"
```

Comment:
add the parameter "$expand": "instanceView" as the user needs the to provide it when calling Get VM with Instance view api.